tests/integration/cqlengine/query/test_queryset.py

```diff
@@ -266,6 +271,10 @@ def setUpClass(cls):
                                         test_result=45, test_list=[1, 2, 3], test_set=set([1, 2, 3]),
                                         test_map={'1': 1, '2': 2, '3': 42})
 
+        IndexedTestModel.objects.create(test_id=14, attempt_id=4, description='list14', expected_result=75,
```

Comment:
We will mostly forgot that many times. I would that whenever we need an id, we use a test class function. ie. test_next_id()